### PR TITLE
Japi 332 WsClient update with ESP 7.12.x

### PR DESCRIPTION
**Pull request by rpastrana**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [ ] The change has been fully tested:
  - [ ] I have performed unit tests to cover my changes.
  - [ ] I have performed system test and covered possible regressions and side effects.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.

### CODE REVIEW

**Comment by rpastrana** [comment]
@jpmcmu please review

**Comment by rpastrana** [comment]
@drea FYI updating wsclient to include all ESP interface changes. I didn't notice anything that would impact you, but please build against your code and let us know if you encounter any issues, thanks.

**Review by jpmcmu — COMMENTED** [review]
A couple of questions, otherwise looks good

File: wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/wsdfu/DFUDataColumnWrapper.java [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-import org.hpccsystems.ws.client.gen.axis2.wsdfu.v1_51.DFUDataColumn;
+import org.hpccsystems.ws.client.gen.axis2.wsdfu.v1_55.DFUDataColumn;
 import org.hpccsystems.ws.client.platform.DFUDataColumnAnnotation;
 
 // This class wraps the generated soap ECL Workunit, providing comparable and to-string methods for end-users.
 
 public class DFUDataColumnWrapper// extends DFUDataColumnImpl
 {
-    private static final long       serialVersionUID = 1L;
+    private static final long       serialVersionUID = 155L;
```
[the remark this answers is not in the document]
Comment (rpastrana) [reply wsclient/src/main/java/org/hpccsystems/ws/client/wrappers/wsdfu/DFUDataColumnWrapper.java:13]: This actually slipped in -- yes the original thought was to keep the UID in line w/ the ESP version. 
The thought was... a change in the ESP interface implies a change in the generated code, however, not all wrappers are affected by a change in the ESP service. As part of the automation effort, we'll need to programmatically update the serial version based on actual changes...

File: wsclient/pom.xml [hunk 2]
```diff
@@ -182,20 +182,20 @@
...
                    <namespaceURI>
                      <uri>urn:hpccsystems:ws:filespray</uri>
-                     <packageName>org.hpccsystems.ws.client.gen.axis2.filespray.v1_16</packageName>
+                     <packageName>org.hpccsystems.ws.client.gen.axis2.filespray.v1_20</packageName>
                    </namespaceURI>
                  </namespaceURIs>
                </configuration>
              </execution>
              <execution>
-               <id>filespray117</id>
```
[the remark this answers is not in the document]
Comment (rpastrana) [reply wsclient/pom.xml:198]: Not as far as I can tell, we're keeping the 1_17 for archival purposes

**Comment by rpastrana** [comment]
@Michael-Gardner please review pom

**Comment by Michael-Gardner** [comment]
@rpastrana looked over the wsclient pom and everything looks fine.  Built fine as well after the merge and change you made.

**Pull request merged**